Makefile

```diff
@@ -0,0 +1,10 @@
+# Default target
+all:
+    zig build
+
+# Clean target
+clean:
+    rm -rdf ./bin
```

Comment:
Wouldn't this delete game data files if somebody copied them into bin/

Reply:
Yeah, it would. This should be way more selective.